src/openrct2/management/Research.h

```diff
@@ -46,6 +46,13 @@ enum class ResearchCategory : uint8_t
     SceneryGroup = 6,
 };
 
+enum class ResearchStatusType : uint8_t
+{
+    NeverInvented,
```

Comment:
if you structure this as `Uninvented, Invented, NeverInvented` then import code will be much simpler.